docs: Upgrade instructions for 0.17 (#1286)
keptn/keptn.github.io: content/docs/install/access/index.md

This option exposes Keptn externally using a cloud provider's load balancer (if available).

> 1. **Install Keptn:** For installing Keptn on your cluster, please use the Helm CLI.

Reviewer: ```suggestion
1. **Install Keptn:** Use the Helm CLI to install Keptn on your cluster.
```

> Depending on whether you would like to install the execution plane for continuous delivery, add the flag `continuousDelivery.enabled=true`.

Reviewer: ```suggestion
If you want to install the execution plane for continuous delivery, add the flag `continuousDelivery.enabled=true`.
```

This option exposes Keptn on each Kubernetes Node's IP at a static port. Therefore, please make sure that you can access the Kubernetes Nodes in your network.

> 1. **Install Keptn:** For installing Keptn on your cluster, please use the Keptn CLI.

Reviewer: ```suggestion
1. **Install Keptn:** Use the Keptn CLI to install Keptn on your cluster.
```

> Depending on whether you would like to install the execution plane for continuous delivery, add the flag `continuousDelivery.enabled=true`.

Reviewer: ```suggestion
If you want to install the execution plane for continuous delivery, add the flag `continuousDelivery.enabled=true`.
```

### Option 3: Expose Keptn via an Ingress

> 1. **Install Keptn:** For installing Keptn on your cluster, please use the Keptn CLI.

Reviewer: ```suggestion
1. **Install Keptn:** Use the Keptn CLI to install Keptn on your cluster.
```

> Depending on whether you would like to install the execution plane for continuous delivery, add the flag `continuousDelivery.enabled=true`.

Reviewer: ```suggestion
If you want to install the execution plane for continuous delivery, add the flag `continuousDelivery.enabled=true`.
```

This option does not expose Keptn to the public but exposes Keptn on a *cluster-internal* IP.

> 1. **Install Keptn:** For installing Keptn on your cluster, please use the Keptn CLI.

Reviewer: ```suggestion
1. **Install Keptn:** Use the Keptn CLI to install Keptn on your cluster.
```

> Depending on whether you would like to install the execution plane for continuous delivery, add the flag `continuousDelivery.enabled=true`.

Reviewer: ```suggestion
If you want to install the execution plane for continuous delivery, add the flag `continuousDelivery.enabled=true`.
```
